backend/src/services/ProgressService.ts

```diff
@@ -41,14 +41,6 @@ class ProgressService {
       return `Cannot advance; application is already ${progress.state}`;
     }
 
-    const currentStage = await StageService.getByPipelineAndIndex(pipeline, progress.stageIndex);
```

Comment:
I removed this "all reviews complete before advancing application" validation because in the past, designers have just given VP Tech a list of candidates to advance/reject, but haven't entered their reviews into Fulcrum. I think VP Tech should have the power to advance/reject people without this guardrail since they can clearly see on the UI whether reviews are complete